TESTING.MD

```diff
@@ -0,0 +1,116 @@
+---
+weight: 121
+toc: true
+title: Tests during development
+menu:
+    docs:
+        parent: contributing
+lead: ""
+lastmod: "2023-09-10T08:48:57+00:00"
+images: []
+draft: false
+description: Understanding and running tests when developing Prometheus-Operator
+date: "2023-09-10T00:00:00+00:00"
```

Comment:
Not sure about this part. Should we put this on the website?

Reply:
I don't think so.
It will be referenced on the CONTRIBUTING and other files in same format are not part of the website

Reply:
I don't see why it couldn't be in the website eventually but it's ok for me to not include it there in the first iteration.

Reply:
I've removed the headline for now